workflows/sarek/main.nf

```diff
@@ -874,7 +874,7 @@ workflow SAREK {
                 vcf_to_annotate.map{meta, vcf -> [ meta + [ file_name: vcf.baseName ], vcf ] },
                 vep_fasta,
                 params.tools,
-                params.snpeff_genome ? "${params.snpeff_genome}.${params.snpeff_db}" : "${params.genome}.${params.snpeff_db}",
+                params.snpeff_db,
```

Comment:
Should we add some early failures when we do annotation and no snpeff_db is specify?
And similarly for VEP?

Reply:
Sounds like a good idea. I got a difficult-to-understand error msg when I (unintentionally) had specified `snpeff_genome` instead of `snpeff_db` in my nf-cmd.

Reply:
Added some error handling